docs/source/standard.md

```diff
@@ -113,65 +113,130 @@ The ability of the TRE organisation to provide and manage devices, workspaces, i
+    This may include version control software, and tools for developing and running data analysis pipelines.
+* - A TRE may provide shared services that are accessible to users in the same project.
+  - This may include shared file storage, databases, collaborative writing, and other web applications.
+    This must only be shared amongst users within the same project.
+* - A TRE may provide limited access to some software repositories
+  - For example, a TRE may allow installation of packages from Python or R repositories, or provide an internal mirror with approved packages.
+```
+
+#### 2.1.3 High performance or cluster computing
+
+The ability to run analyses requiring more compute resources than is present in the user's workspace.
+
+```{list-table}
+:header-rows: 1
+:name: tab-end-user-high-performance-cluster-computing
+* - Statement
+  - Guidance
+* - A TRE should provide access to high performance computing or other scaleable compute resource if required by users.
+  - If a TRE supports users conducting computationally intensive research it should provide access to dynamically scaleable compute or the equivalent.
+    For example this may be in the form of a batch scheduler on a HPC cluster, or a dynamically created compute nodes on a cloud platform.
+    Users from different projects must not have access to the same compute nodes.
+    When using physical compute resources all sensitive data must be securely wiped before another user is given access to that same node.
+```
 
 #### 2.1.4 Accelerators
 
-GPU, FPGA, ASIC, xPU
-
-| Statement | Guidance |
-| --------- | -------- |
-|           |          |
+The ability to provide accelerators such as GPUs
 
-#### 2.1.5 Cluster computing
+```{list-table}
+:header-rows: 1
+:name: tab-end-user-accelerators
+* - Statement
+  - Guidance
+* - A TRE should provide access to accelerators such as GPUs if required by users.
+  - GPUs and pother accelerators are commonly used in machine learning and other computationally intensive research.
+    TREs should make it clear to users whether GPUs and other resources are available whilst projects are being assessed.
+```
 
-SLURM, Kubernetes etc.
-
-| Statement | Guidance |
-| --------- | -------- |
-|           |          |
-
-#### 2.1.6 Databases
+#### 2.1.5 Databases
 
+Provision of databases for users
 SQL, noSQL, etc.
 
-| Statement | Guidance |
-| --------- | -------- |
-|           |          |
+```{list-table}
+:header-rows: 1
+:name: tab-end-user-software-tools
+* - Statement
+  - Guidance
+* - A TRE may make data available to researchers using comonly used databases such as PostgreSQL, MSSQL, MongoDB, etc.
+  - Databases must be secured and only accessible to users within the same project.
+    If shared database servers are used database administrators must ensure the database enforces segregation of users.
```

Comment:
I feel like the second contradicts the first. "Databases must be restricted to a particular project. But if they are not..."